src/main/webapp/scripts/script.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
Delete this entire file. Don't check in empty files. Only checking files with actual substance